docs/guides/upgrading.rst

```diff
@@ -124,6 +124,13 @@ API to alter registration and login URL
  * ``registration_url, site`` hook can be used to alter the default registration URL
  * ``login_url, site`` hook can be used to alter the default login URL
 
+Support for fieldsets in forms
+------------------------------
+
+ * ``elgg_view_input()`` now supports ``#type``, ``#label``, ``#help`` and ``#class`` parameters to prevent name collision with input view ``$vars``
```

Comment:
I guess I think it might be less clear reading form code where both `#class` and `class` are keys doing different things.